### ISSUE TITLE
HomeGenie Android denies a testdrive, frustrates users

### ISSUE THREAD

**Issue by pfalcon** [genielabs/HomeGenie-Android#3]
I was very excited to see beautiful screenshots of HomeGenie Android app, so I went to Play Store and installed it (no, I don't have HomeGenie itself installed yet - I just want to give an Android client a testdrive). When I start it, it greets me with stupid dialog to connect to server (which doesn't even have "Ok" button, so I'm not sure it works at all) and does not let me in.

At this point, I'm getting frustrated, so if I were a mere user, I'd get back to Playstore to give a one star with comment "doesn't work at all!!1111" (note: I am not a mere user, but open-source author/contributor, so I appreciate any FOSS project and would never frustrate an author with one star).

Bottom line: please let users to have a testdrive w/o server installed. I'm sure that at least weather forecast you can show them w/o server ;-). Even if everything in client depends on getting info from server, please consider adding a pre-loaded sample static configuration.

Thanks!

**Comment by bkenobi** [comment]
This is a client software that is used to connect to the server.  How can you test drive a client without having a server set up?

**Comment by pfalcon** [comment]
Exactly like you write - testdrive without connect to a server.

More specifically, what connection to server does is loads sensor/actuator config from it. And surely, HomeGenie *server* comes with a sample device config to let people play with it immediately after install, before spending time to configure for "production" use. The same sample config can go into Android client, plus a button "Demo" and couple of boolean checks for this demo mode (to not try to send/receive to unconfigured server).

If concept sounds still too novel and hard to believe, have a look at [FHEM Android client](http://andfhem.klass.li/), which worked like that for ages.

**Comment by genielabs** [comment]
@pfalcon I understad your point, but still it's preferable encouraging people to install HG server. There's so much behind it to discover than just turning things on and off. Installing HG server it just takes a minute.
